parm/archive/gefs_atmos.yaml.j2

```diff
@@ -0,0 +1,45 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs:
+    name: "GEFS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_atmos.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set file_set = [] %}
+{% set membr = ["ensstat", "mem000", "mem001", "mem002"] %}
```

Comment:
Instead of setting the list manually, I tried this.
`{% set membr = ["ensstat"] %}
{% for mem_nm in range(0, NMEM_ENS + 1) %}
    {% do membr.append("mem{{ '%03d' % mem_nm }}") %}  
{% endfor %}`

In this membr does not append and produce ["ensstat", "mem000", "mem001", "mem002"].  Any suggestions?

Reply:
When a statement is within `{% %}`, `{{ }}` should not be used.  Everything within `{% %}` is assumed to be a variable name unless it is in quotes.  So I think you want this:

```jinja
{% set membr = ["ensstat"] %}
{% for mem_nm in range(0, NMEM_ENS + 1) %}
    {% do membr.append("mem" ~ '%03d' % mem_nm ) %}
{% endfor %}
```

Reply:
Thanks